release_notes/ocp-4-11-release-notes.adoc

```diff
@@ -124,6 +124,13 @@ For more information, see xref:../installing/installing_azure/installing-azure-c
 
 This update introduces a new option for Ingress Controllers with the `hostnetwork` endpoint strategy. You can now host multiple Ingress Controllers on the same worker node using `httpPort`, `httpsPort`, and `statsPort` binding ports.
 
+[id=ocp-4-11-multiple-configuration-control-worker-nodes]
+==== Multi-node configuration for controller and worker nodes
+
+When you configure nodes, you can use a mechanism that applies a single configuration to multiple, bare-metal, installer-provisioned infrastructure (IPI) nodes in a cluster. Applying a single configuration that defines multiple nodes in your cluster reduces the risk of misconfiguration due to the single-provisioning process.
```

Comment:
Can shorten the sentence by replacing 'that defines' to 'to'.

Reply:
Done